### PR TITLE
fuzz: fix title caps in options panel

### PR DESCRIPTION
**Pull request by njmulsqb**
Fixed the title caps in Fuzzer options screen

Related to zaproxy/zaproxy#2000

### CODE REVIEW

**Review by kingthorin — APPROVED** [review]
This seems good to me. It aligns with how things are formatted in the Active Scan panel.

Were you intending to make further changes, is that why it's marked WIP?

**Comment by kingthorin** [comment]
Was this one done via the GitHub Web UI?

**Comment by thc202** [comment]
It was me who set it to WIP, since the help is being changed at the same time and both should match.

**Comment by thc202** [comment]
The changelog should be updated too.

**Comment by njmulsqb** [comment]
> Was this one done via the GitHub Web UI?

Yes

**Comment by kingthorin** [comment]
I've squashed the commits and tweaked the changelog entry.

**Comment by thc202** [comment]
There's a typo in the commit message (note).

File: addOns/fuzz/CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Now using 2.10 logging infrastructure (Log4j 2.x).
 - Maintenance changes.
 - Update dependency (Issue 4751).
+- Fixed title caps in messages.properties (Issue 2000).
```
Comment (thc202) [inline addOns/fuzz/CHANGELOG.md:18]: Needs to be moved to Unreleased.

Comment (kingthorin) [reply addOns/fuzz/CHANGELOG.md:18]: I'll tackle this.

**Comment by kingthorin** [comment]
Done & done

**Comment by thc202** [comment]
Addressed the conflict.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you both!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**